packages/appium/docs/zh/contributing/develop.md

```diff
@@ -76,7 +76,7 @@ Our documentation system uses [MKDocs](https://www.mkdocs.org/) and therefore re
 
 ```sh
 # installing needed Python dependencies
-pip install -r packages/docutils/requirements.txt --break-system-packages
+pip install -r packages/docutils/requirements.txt
```

Comment:
should we also add the env var prefix to this doc?

Reply:
I guess we could. This doc as a whole is a bit outdated compared to the `en` version, which uses `npm run install-docs-deps`.